filaments/sunlu.json

```diff
@@ -1,6 +1,28 @@
 {
     "manufacturer": "Sunlu",
     "filaments": [
+        {
+            "name": "{color_name}",
+            "material": "PETG",
+            "density": 1.23,
+            "weights": [
+                {
+                    "weight": 1000.0,
+                    "spool_weight": 159.0
+                }
+            ],
+            "diameters": [
+                1.75
+            ],
+            "extruder_temp": 235,
+            "bed_temp": 70,
+            "colors": [
+                {
+                    "name": "Elite Series {color_name}",

```

Comment:
You can't use the color_name tag here, I think you perhaps wanted to put this in the name field on line 5.

Here you should type in the name of the color itself.